### PR TITLE
refactor!: standardise font sizes across all screen sizes

### PR DESCRIPTION
**Pull request by helendurranttw**
## 🖼 Context

We've decided to standardise at font 16px across all screen sizes (and platforms!).

## 🚀 Changes

Removed the 15px body font size at small screen sizes.

## 🤔 Considerations

This is a breaking change - we're aware some people may make adjustments at small screen sizes for the smaller font size.

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

**Review by andreapiras — DISMISSED** [review]

File: packages/css/src/less/core/_scaffolding.less [hunk 1]
```diff
@@ -246,19 +241,11 @@ ul {
 
 dt,
 dd {
-  line-height: @line-height-base;
-
-  @media (--screen-md) {
-    line-height: @line-height-base--desktop;
-  }
+  line-height: @line-height-small;
```
Comment (andreapiras) [inline packages/css/src/less/core/_scaffolding.less:244]: image
Should this be? `@line-height-base`
Would this not result in 1.6 * 16px line-height -> 25.6px ?

File: packages/css/src/less/core/_typography.less [hunk 2]
```diff
@@ -158,20 +146,12 @@ small,
 .small {
   font-size: @font-size-small;
   line-height: 1.7142857143; // 24/14
```
Comment (andreapiras) [inline packages/css/src/less/core/_typography.less:148]: This can be @line-height-small

File: packages/css/src/less/core/_typography.less [hunk 3]
```diff
@@ -239,10 +219,6 @@ blockquote {
     font-size: @font-size-small;
     line-height: 1.7142857143; // 24/14
```
Comment (andreapiras) [inline packages/css/src/less/core/_typography.less:220]: This can be @line-height-small

File: packages/css/src/less/core/_typography.less [hunk 4]
```diff
@@ -254,7 +230,7 @@ blockquote {
 
   @media (--screen-lg) {
     margin: var(--size-32);
-    font-size: @font-size-medium--desktop;
+    font-size: @font-size-medium;
     line-height: 1.3333333333; // 24/18
```
Comment (andreapiras) [inline packages/css/src/less/core/_typography.less:234]: Maybe we should introduce `@line-height-medium` ?

File: packages/css/src/less/core/_typography.less [hunk 5]
```diff
@@ -68,35 +68,23 @@ h4,
...
+  font-size: @font-size-base;
+  line-height: @line-height-base; // 24/16px
 }
 
 h4,
 .h4 {
   @media (--screen-md) {
-    font-size: @font-size-medium--desktop;
+    font-size: @font-size-medium;
     line-height: 1.3333333333; // 24/18
```
Comment (andreapiras) [inline packages/css/src/less/core/_typography.less:79]: Maybe we should introduce @line-height-medium ?

File: packages/css/src/less/core/_typography.less [hunk 6]
```diff
@@ -68,35 +68,23 @@ h4,
...
-    font-size: @font-size-base--desktop;
-    line-height: @line-height-base--desktop;
-  }
-}
-
 h6,
 .h6 {
   color: @color-base-grey-light;
   font-size: @font-size-small;
   line-height: 1.7142857143; // 24/14
```
Comment (andreapiras) [inline packages/css/src/less/core/_typography.less:87]: Maybe we should introduce @line-height-large ?

File: packages/css/src/less/core/_typography.less [hunk 7]
```diff
@@ -149,7 +137,7 @@ p {
   line-height: 1.7777777778; // 32/18
 
   @media (--screen-md) {
-    font-size: @font-size-large--desktop;
+    font-size: @font-size-large;
     line-height: 1.4545454545; // 32/22
```
Comment (andreapiras) [inline packages/css/src/less/core/_typography.less:141]: Maybe we should introduce @line-height-large ?

**Review by anttio — APPROVED** [review]

**Review by andreapiras — APPROVED** [review]

**Pull request merged**